docs/general/cloud-events.md

```diff
@@ -286,6 +286,29 @@ The application then may get the latest terms of use through the [Get Terms Of U
 }
 ```
 
+## Consent update events
+
+Fired whenever a user grants or revokes a consent. A whole deletion of a consent triggers a revoked event.
+Applications that are part of any of the users subscriptions receive a notification. The application can then retrieve the latest consense status of the user.
+
+`tapio.consentinator.consent.revoked`
```

Comment:
The cloud event type should not expose any reference to internal services. The event type should start with `one.tapio.selfservice`.

Reply:
According to our discussions in the integration meeting, start should be `tapio.selfservice`
See [internal docs](https://internaldocs.tapio.one/Development-Guidelines/CloudEventType#common-schema).